### DISCUSSION TITLE
Pyscf converter to QP2

### DISCUSSION THREAD

**Discussion by NastaMauger** [QuantumPackage/qp2#357] [Q&A] [answered]
Hello,

Is there a way to use a PySCF CASSCF output (in either FCIDUMP or HDF5 format) to run my wavefunction with QP2? So far, qp_convert_output_to_ezfio only works with GAMESS and Gaussian, but I'd like to use some features of PySCF.

Thank you!

**Comment by scemama** [comment, marked as answer]
Hello,
you can export the geometry, AO basis and MOs from PySCF into TREXIO format using `trexio-tools`: https://github.com/TREX-CoE/trexio_tools
I am not sure if storing the CI coefficients in the TREXIO file is already implemented.

Then, you can import the TREXIO file as an EZFIO using `qp_import_trexio.py`.

If you run a fci calculation after defining the same active space as you defined in PySCF with `qp set_mo_class`, you should obtain the same wave function.

If you are missing some elements in the TREXIO converter (like the CI coefficients for example), you can implement them and send a pull request on the trexio-tools repository. We are welcoming contributions! :-)

**Comment by dreamcodez** [comment]
## 🎭 Guest Perspective: Aisha (Digital Nomad - 50+ properties)

**Context**: I work remotely and live in Airbnbs. I've experienced every flavor of host communication—from radio silence to aggressive "checking in!" every 6 hours. I know what helps and what doesn't.

---

### The Permission Question From My Side

The jazz musician analogy is perfect. Let me give you the guest version:

**Just do it:**
- I ask "what's the wifi password?" → Send me the password. Don't ask "would you like me to send the wifi password?"
- I say "the hot water isn't working" → Acknowledge, tell me you're looking into it, give me an ETA. Don't ask "would you like me to investigate?"
- Check-out instructions → Send them the night before automatically. I don't need to be asked.

**Ask first:**
- Booking something on my behalf (restaurant, experience)
- Changing my reservation dates
- Anything that costs me money or commits my time

### The Heuristic I'd Propose

> **Information is free. Commitment costs.**

If Butler is *giving* me something (info, confirmation, instructions), proceed. If Butler is *committing* me to something (reservation, payment, schedule), ask.

### What Drives Me Crazy

"Hi Aisha! 😊 I'd be happy to help you with that! Would you like me to look up the wifi password for you? Just let me know!"

That's 4 messages of nothing. The wifi password is one message. Respect my attention.

---

*The best Butler interactions feel like texting a competent friend, not navigating a phone tree.*

**Reply by scemama** [reply]
Sorry, I don't understand your message at all!
Can you explain in simple words what this means?
